man/rgbgfx.1

```diff
@@ -103,7 +103,9 @@ Same as
 .Fl a Ar path ,
 where
 .Ar path
-is the input image's path with the extension set to
+is the input image's path
+.Pq or the output tile data path from Fl o No if Fl O No was specified
```

Comment:
Perhaps we should centralise all copies of this description? And coin a term for them, like "default path" or "automatic path"?

This would allow for a more detailed description, which I feel would be helpful.

Reply:
How about this: for all four of those options, we describe them as such:

> <code>-<i>W</i>, --output-<i>whatever</i></code>
> Same as <code>-<i>w</i></code> path, where <i>path</i> is the base path (see "Automatic output") with the extension set to .<i>whatever</i>.

Then in the "Output files" section, we have a subsection "Automatic input", which describes how the base path can be either the input image or the output tile data path, depending on `-O`.

The questions are, what to call that section besides "Automatic input" ("Auto-named output"? "Automatic output"? "Derived output"?), and what to call the base path (original path? fundamental path? path template? source path?).

Reply:
Here's my take:

```man
.It Fl W , Fl Fl output-whatever
Same as
.Fl w Ar base_path Ns .whatever
.Pq see Sx Automatic output paths .
.\" ...
.Ss Automatic output paths
For convenience,
.Nm
provides shortcuts to generate all files in the same directory: this is done by using the uppercase version of a flag
.Pq for example, Fl A No instead of Fl a .
The
.Ar base_path
is the input image's path
.Pq or the path passed to Fl o , No if Fl O No was given
with its extension, if any, removed.
.Pp
For example, these two are equivalent:
.Bd -literal -indent Ds
$ rgbgfx img/player_anim.png -o gfx/player.png -P
$ rgbgfx img/player_anim.png -o gfx/player.png -p img/player_anim.pal
.El
.Pp
.Bd -literal -indent Ds
$ rgbgfx -O img/player_anim.png -o gfx/player.png -P
$ rgbgfx -O img/player_anim.png -o gfx/player.png -p gfx/player.pal
.El
.Pp
.Fl O
is intended for out-of-tree builds, as can be seen above.
```

Additionally, should we consider renaming `--output-*` to `--auto-*` or `--*-auto`? That seems clearer to me. (That said, we'd keep the old names for compatibility's sake. With or without a deprecation warning.)

Reply:
Let's do a separate issue+PR for `--auto-*` renaming. I've pushed the "Automatic output paths" section.

Reply:
Filed as #1243.